ios/README.md

```diff
@@ -1,5 +1,28 @@
-## LibTorch
 
-The PyTorch C++ static library for iOS. 
+## LibTorch for iOS
 
-(Detailed documentation  will be added soon)
\ No newline at end of file
+### Cocoapods Developers
+
+PyTorch is now available via Cocoapods, to integrate it to your project, simply add the following line to your `Podfile` and run `pod install`
+
+```ruby
+pod 'LibTorch'
+```
+
+### Import the library
+
+For Objective-C developers, simply just import the umbrella header
+
+```
+#import <LibTorch/LibTorch.h>
```

Comment:
How come it's `LibTorch/LibTorch.h`? In C++, it's just `torch/torch.h`, why is it different here?

Reply:
`LibTorch.h` is reserved as an umberlla header that should be imported by Objective-C code. 
Later we'll have Objective-C / Swift wrapper code, which will  rely on this umbrella header to create a swift module.

Reply:
Is there some sort of Objective-C convention here that I don't know about? :)

Reply:
Sorry, didn't see the reply. How to turn on the notification bell on Github? Regarding the umbrella header, I found an article - http://blog.cocoapods.org/Pod-Authors-Guide-to-CocoaPods-Frameworks/. Hopefully, it'll be useful.

Reply:
check your notifications at https://github.com/notifications

by default GitHub will "watch" a bunch of repositories for you which is very bad; unwatch from high traffic ones so you only get pings on explicit mentions and issues you're subscribed to